apps/django_app.py

```diff
@@ -20,5 +21,24 @@
 filename = os.path.basename(os.path.splitext(__file__)[0])
 
 if __name__ == "__main__":
-    settings.configure(ROOT_URLCONF=filename, ALLOWED_HOSTS="localhost")
+
+    options = {
+        "ROOT_URLCONF": filename,
+        "ALLOWED_HOSTS": ["localhost", "127.0.0.1", "[::1]"],
+    }
+
+    if os.environ.get("VULNPY_USE_CONTRAST"):
```

Comment:
is the `VULNPY_` prefix a good idea in case other env vars are called `USE_CONTRAST`? Just a thought to not prefix with vulnpy, but I don't mind either way.

Reply:
I guess this is just personal preference, but I like making env vars pretty specific just in case. But yes, that is the only reason why I have `VULNPY_` in there